### ISSUE TITLE
Including links and images

### ISSUE THREAD

**Issue by 1jamie7** [mHaisham/novelsave#30]
Hi! 🤗 Is there a way for the links included in the websites to be also active when the epubs are made? The links don't work in the epubs even though they work on the websites. 🥺
Thank you so much in advance! ❤❤

**Comment by mHaisham** [comment]
If you could point me towards a specific, it would make testing easier

**Comment by 1jamie7** [comment]
At the bottom of this https://chrysanthemumgarden.com/novel-tl/pubg/pubg-26/ chapter

from the site

![20210505_212616.jpg](https://user-images.githubusercontent.com/74856769/117149432-124c5e00-ad6c-11eb-847d-1baccdfb4ce3.jpg)


the epub i made

![20210505_213028.jpg](https://user-images.githubusercontent.com/74856769/117149544-298b4b80-ad6c-11eb-9f87-96cf7131e6fd.jpg)

**Comment by mHaisham** [comment]
link element <a> is supported in epub. during the cleaning process of the chapter `href` (actual link, ex: `<a href="https://" ></a>`) of the element is being removed.

Fixing the link problem should be as simple as fixing the cleanup process.

However, i'm afraid that the <img> situation is a bit more complex.

epub's are offline documents and shoudnt request data online, this means that the images need to be downloaded and packed into the epub. While this sounds simple, the problem arises from how i dont have a system in place for this.

I do plan on writing such a system, however it might take sometime.

**Comment by mHaisham** [comment]
I tested using application ReadEra, It doesn't seem to show links. So I read about it online and it seems that different 
reading applications handle links differently.

**Comment by 1jamie7** [comment]
Yes, I used another epub reader too and it only shows the highlighted link but it is still not working..


this is from Lithium epub reader


![20210506_051629.jpg](https://user-images.githubusercontent.com/74856769/117210883-23698f00-adad-11eb-9c7d-22b262726fc1.jpg)

**Comment by 1jamie7** [comment]
About the system you mentioned above, that would be great 🤧 I can wait, take your time! 😍😍

**Comment by 1jamie7** [comment]
Hi! The links worked on my two epub reader now. 🥳😍😍Thank you so much! 😭😭❤❤❤❤

**Comment by mensch272** [comment]
Images will finally be added in #35
![image](https://user-images.githubusercontent.com/47662901/130682042-2932d845-8caa-495d-8f7c-80de34ee2c2c.png)

**Comment by 1ohr** [comment]
hi! 🤗 thank you so much for all the efforts you make to make this possible 😭😭😭 i really really appreciate this!  🤧🤧🤧 thank you ❤❤❤

by the way, i am the person who created that issue before 😁 i just deleted that account of mine 😅
thank you again 😭😭😘😘😘
